### PR TITLE
CAT-1248 Cursor Design

### PR DESCRIPTION
**Pull request by obusher**
Reopening of #1157 from @Achilles-96. :palm_tree:

### CODE REVIEW

File: catroid/src/org/catrobat/catroid/formulaeditor/FormulaEditorEditText.java [hunk 1]
```diff
@@ -40,12 +41,13 @@
 public class FormulaEditorEditText extends EditText implements OnTouchListener {
 
 	private static final BackgroundColorSpan COLOR_ERROR = new BackgroundColorSpan(0xFFF00000);
-	private static final BackgroundColorSpan COLOR_HIGHLIGHT = new BackgroundColorSpan(0xFFFFFF00);
+	private static final BackgroundColorSpan COLOR_HIGHLIGHT = new BackgroundColorSpan(0xff33b5e5);
```
Comment (aried3r) [inline catroid/src/org/catrobat/catroid/formulaeditor/FormulaEditorEditText.java]: Have we decided on whether to use uppercase or lowercase for hex values? I think we use only uppercase in the `colors.xml`, at least I advocate on doing so.

**Comment by Achilles-96** [comment]
Been a long time. Would you merge my commits?

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by avihric** [comment]
@obusher Commit msg looks strange, also amount

**Comment by obusher** [comment]
This PR only contains now CAT-1248. Commit message got rewritten.

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by avihric** [comment]
Retest `testRenameSpriteDialogMixedCase` and squash the commits and I am fine to merge :smile:

**Comment by obusher** [comment]
I will not squash the commits because the first one does not belong to me. 
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-Device-Single-ClassAndPackage-Test/277/

**Comment by avihric** [comment]
But how does this commit show up in your PR then. I suppose that this was
added during rebase, therefor it should not be in this PR imho.

**Comment by obusher** [comment]
Nope it was not added during rebase, it got `cherry-picked` and it should be in this PR.

**Comment by avihric** [comment]
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/388/

**Pull request merged**